### PR TITLE
tg-can-668

### PR DESCRIPTION
**Pull request by nalwa**
This PR has minor fix required from API end, rest will be fixed from fron-end.

@akuks  please review and merge PR

### CODE REVIEW

File: app/Models/Camp.php [hunk 1]
```diff
@@ -453,7 +453,11 @@ public static function getDirectCampSupporterIds($topicid, $campnum)
     public static function validateParentsupport($topicNum, $campNum, $userNicknames) 
     {
         $filter = self::getLiveCampFilter($topicNum, $campNum);
-        $oneCamp = self::getLiveCamp($filter);        
+        $oneCamp = self::getLiveCamp($filter);   
+
+        if(empty($oneCamp)){
```
Comment (akuks) [inline app/Models/Camp.php:458]: `if(!$onecamp)` will ot work

File: app/Models/Camp.php [hunk 2]
```diff
@@ -453,7 +453,11 @@ public static function getDirectCampSupporterIds($topicid, $campnum)
...
+            return 'notfound';
```
Comment (akuks) [inline app/Models/Camp.php:459]: May be return 0 or false instead of string?

**Review by akuks — CHANGES_REQUESTED** [review]

File: app/Models/Camp.php (hunk 2 above)
Comment (nalwa) [reply app/Models/Camp.php:459]: Zero or false will not work as False or Zero  means no support, but its the case of not found record.

File: app/Models/Camp.php (hunk 1 above)
Comment (nalwa) [reply app/Models/Camp.php:458]: Not sure because sometimes it gives empty eloquent model which does not work  with !

**Review by akuks — APPROVED** [review]

**Pull request merged**